src/state/lists/reducer.test.ts

```diff
@@ -386,18 +386,19 @@ describe('list reducer', () => {
 
       it('each of those initialized lists is empty', () => {
         const byUrl = store.getState().byUrl
-        // note we don't expect the uniswap default list to be prepopulated
-        // this is ok.
-        Object.keys(byUrl).forEach((url) => {
-          if (url !== 'https://unpkg.com/@uniswap/default-token-list@latest/uniswap-default.tokenlist.json') {
-            expect(byUrl[url]).toEqual({
+        Object.entries(byUrl)
+          // We don't expect the Uniswap default list to be prepopulated
+          .filter(
+            ([key]) => key !== 'https://unpkg.com/@uniswap/default-token-list@latest/uniswap-default.tokenlist.json'
+          )
+          .forEach(([, state]) => {
```

Comment:
`value`, not `state`, because you're using `entries()`

Reply:
I guess it's just a personal preference - I often rename `key` and `value` to give it more meaningful names, such as `url` and `state` in this case.

It looks like both patterns are currently present in the codebase, e.g.:
`Object.entries(MARKETPLACE_ITEMS).map(([value, title])`
or
`Object.entries(traitsByGroup).map(([type, traits]))`

Reply:
> I often rename key and value to give it more meaningful names

This is better - I agree - but in this case you're being inconsistent. You should either rename `key` to `url`, or `state` to `value`.

Reply:
Right 👀 I will update `filter`, don't know why I skipped it while glancing the code.